src/SignalSlot/RoleService/DeleteRoleSlot.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+/**
+ * This file is part of the eZ Publish Kernel package.
+ *
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+namespace EzSystems\PlatformHttpCacheBundle\SignalSlot\RoleService;
+
+use eZ\Publish\Core\SignalSlot\Signal;
+
+/**
+ * A slot handling delete role signals.
```

Comment:
I think that this approach is more readable:
https://github.com/ezsystems/ezplatform-http-cache/pull/59/files#diff-2469f72d42b2f86cd6e318f5ba82b4dfR12 or https://github.com/ezsystems/ezplatform-http-cache/pull/59/files#diff-88acad2253affe57926847b2b01ec0fbR13
so `A slot handling DeleteRoleSignal` instead of `A slot handling delete role signals.`